05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+1. Divide the sample into subsample **replicates** that mirror the design of the sample
+2. Calculate weights for each **replicate** using the same procedures for full-sample weight (i.e. nonresponse and post-stratification)
+3. Calculate estimates for each **replicate** using the same method as the full-sample estimate
+4. Calculate the estimated variance which will be proportional the the variance of the replicate estimates
+
+The different types of replicate weights largely differ in step 1 - how the sample is divided into subsamples and step 4 - different multiplication factors (scales) are used to multiply the variance.
+
+### BRR method
+The BRR method requires a stratified sample design with two PSUs in each stratum. Each replicate is constructed by deleting one PSU per stratum using a Hadamard matrix. For the PSU that is included, the weight is generally multiplied by 2 but may have other adjustments such as post-stratification. A Hadamard matrix is a special square matrix with entries of +1 or -1 with mutually orthogonal rows. Hadamard matrices must have 1 row, 2 rows, or a multiple of 4 rows. An example of a $4\times4$ Hadamard matrix is below:
+
+$$ \begin{array}{rrrr} +1 &+1 &+1 &+1\\ +1&-1&+1&-1\\ +1&+1&-1&-1\\ +1 &-1&-1&+1  \end{array} $$
+The columns specify the strata and the rows the replicate. In the first replicate, all the values are +1 so in each of the stratum, the first PSU would be used in the estimate. In the second replicate, the first PSU would be used in stratum 1 and 3 while the second PSU would be used in stratum 2 and 4. In the third replicate the first PSU would be used in stratum 1 and 2 while the second PSU would be used in stratum 3 and 4. Finally, in the third replicate, the first PSU would be used in stratum 1 and 4 while the second PSU would be used in stratum 2 and 3.
+
+#### The math {-}
+
+A weighted estimate for the full sample is calculated as $\hat{\theta}$ and then a weighted estimate for each replicate is calculated as $\hat{\theta}_r$ for $R$ replicates. Then the standard error of the estimate is calculated as:
+
+$$se(\hat{\theta})=\sqrt{\frac{1}{R} \sum_{r=1}^R \left( \hat{\theta}_r-\hat{\theta}\right)^2}$$
+Specifying replicate weights in R requires specifying the type of replicate weights, the main weight, the replicate weights, and some other options. One of the key options is for `mse`. If `mse=TRUE`, variances are computed around the point estimate whereas if `mse=FALSE`, variances are computed around the mean of the replicates which looks like this:
+
+$$se(\hat{\theta})=\sqrt{\frac{1}{R} \sum_{r=1}^R \left( \hat{\theta}_r-\bar{\theta}\right)^2}$$ where $$\bar{\theta}=\frac{1}{R}\sum_{r=1}^R \hat{\theta}_r$$
```

Comment:
Is there a way to highlight the difference between this one and the one above.  It's hard to notice the difference between `\hat{\theta}` and `\bar{\theta}` when rendered.

Reply:
I added a bit more clarifying text with both symbols shown.